modules/administration_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -124,14 +146,20 @@ Members of excluded groups can still receive shares, but not create any.
 
 Check this option to enable auto-completion of ownCloud usernames.
 
-=== Restrict enumeration to group members
+==== Restrict enumeration to group members
 
 Check this option to restrict auto-completion of ownCloud usernames to only those users who are members of
 the same group(s) that the user is in.
 
 NOTE: ownCloud does not preserve the mtime (modification time) of directories, though it does update
 the mtimes on files.
 
+=== Extra field to display in autocomplete results
+The autocomplete dropdowns in ownCloud usually show the display name of other users when it is set.
+If it's not set, they show the user ID / login name, I think. As display names are not unique you can run into
```

Comment:
Please remove "I think" from the second sentence. Always best to be sure, one way or the other.